packages/grafana-data/src/types/plugin.ts

```diff
@@ -58,7 +58,7 @@ export interface PluginMeta<T extends KeyValue = {}> {
   info: PluginMetaInfo;
   includes?: PluginInclude[];
   state?: PluginState;
-  alias?: string;
+  aliasIds?: string[];
```

Comment:
reports as a breaking change... but I think OK

Reply:
nit: the camel case for this would be `aliasIDs`